plump-sexp.lisp

```diff
@@ -73,6 +77,13 @@ Alternatively a pathname, stream or string may be passed as well, which will be
 
 (defgeneric serialize (node)
   (:documentation "Serialize the given node into a SEXP form.")
+  (:method ((node xml-header))
+    (list
+     (cons :!HEADER
+           (when (< 0 (hash-table-count (attributes node)))
+             (loop for key being the hash-keys of (attributes node)
+                   for val being the hash-values of (attributes node)
+                   nconc (list (string->name key) val))))))
```

Comment:
```suggestion
                   collect (string->name key) collect val)))))
```

Rather than nconc of a list it is better to use the collect clause twice.